package.json

```diff
@@ -3,61 +3,45 @@
   "version": "0.0.3",
   "scripts": {
     "interactive": "./scripts/interactive.js",
-    "nightwatch": "nightwatch -c ./conf/dev.js",
+    "nightwatch": "nightwatch -c ./build/conf/dev.js",
     "nightwatch-chrome": "nightwatch -c ./conf/dev.js --env CHROME_LATEST",
     "nightwatch-ff": "nightwatch -c ./conf/dev.js --env FIREFOX_LATEST",
     "browserstack": "./scripts/local.runner.js -c ./conf/browserstack-default.js",
     "browserstack-all": "./scripts/local.runner.js -c ./conf/browserstack-chrome.js; ./scripts/local.runner.js -c ./conf/browserstack-firefox.js",
```

Comment:
need to update these commands to eh?

Reply:
I didn’t address these because I think they’ll change / be removed. Will do some more clean up on these to not leave broken window

Reply:
I didn’t address these because I think they’ll change / be removed. Will do some more clean up on these to not leave broken window

Reply:
I didn’t address these because I think they’ll change / be removed. Will do some more clean up on these to not leave broken window